schemas/osbuild2.json

```diff
@@ -83,6 +86,24 @@
         }
       }
     },
+    "metadata": {
+      "title": "Metadata information for a manifest",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "generator": {
```

Comment:
(super nitpick) if it's an array maybe use plural here, i.e. `generators` ?

P.S. I'm not 100% sold on the multiple generators but I don't mind much either :)